text/0265-event-vision.md

```diff
@@ -0,0 +1,74 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
```

Comment:
```suggestion
The introduction of Events has been contentious, so we want to write down and agree on a few basics.
```

Nit: I would word it as "has created confusion" or "is a source of constant confusion".. contentious has a bad connotation, like there was a heated argument/conflicts. But I'm not a native speaker so maybe it's all fine and just different cultural views :)

Reply:
> like there was a heated argument/conflicts

there have been 😄